pkg/report/report.go

```diff
@@ -798,6 +792,11 @@ func handleOverrides(original, override []*malcontent.Behavior, minScore int) []
 
 	modified := make([]*malcontent.Behavior, 0, len(behaviorMap))
 	for _, b := range behaviorMap {
+		// if running a scan and using quantityIncreasesRisk,
+		// append every behavior so we can handle filtering correctly
+		if scan && quantityIncreasesRisk && b.RiskScore >= 3 {
```

Comment:
```suggestion
		if scan && quantityIncreasesRisk && b.RiskScore >= HIGH {
```

Reply:
We have an enum defined in report.go, we should probably use it consistently, and ideally we'd define the levels globally somewhere, so that we don't have hardcoded integer values all over the place. Fixing that is not a blocker for this PR, though.